docs/UserGuide/Appendix/SQL-Reference.md

```diff
@@ -728,10 +746,10 @@ andExpression : predicate (OPERATOR_AND predicate)*
 predicate : (suffixPath | fullPath) LIKE stringLiteral
 stringLiteral : SINGLE_QUOTE_STRING_LITERAL | DOUBLE_QUOTE_STRING_LITERAL
 
-Eg. select s1 from root.sg.d1 where s1 like 'Regex'
-Eg. select s1, s2 FROM root.sg.d1 where s1 like 'regex' and s2 like 'Regex'
-Eg. select * from root.sg.d1 where s1 like 'Regex'
-Eg. select * from root.sg.d1 where s1 like 'Regex' and time > 100
+Eg. select * from root.sg.d1 where s1 like 'String'
```

Comment:
Why all examples are `select *`, especially you removed examples like `select s1`, `select s1, s2`. Is this kind of statement not supported?